app/views/mailings/_details.html.slim

```diff
@@ -0,0 +1,19 @@
+h1 class="mailing" = @mailing.subject
```

Comment:
I haven't been involved in Rails view code in some time, so please correct me if I'm wrong, but aren't partials supposed to work on "normal" variables rather then view-instance variables? As in one would pass `mailing: @mailing` and then have `mailing` available in the partial.